### PR TITLE
CORE-16283: Resolve the notary cpb for workflow tests using the csde-gradle-plugin

### PR DESCRIPTION
**Pull request by tlawson3**
The workflows tests using the driver require the notary CPB
We already get the CPB using the csde-gradle-plugin, so go look for the artifact on the filesystem.

Tested: https://ci02.dev.r3.com/job/QA/job/Craft_Tests/job/qa-regpack-csde/job/main/385/

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -16,6 +16,7 @@ allprojects {
     def javaVersion = VERSION_17
 
     // Configure the CSDE
+    apply plugin: 'net.corda.plugins.csde'
```
Comment (marvincudjoe) [inline build.gradle]: lgtm but it's missing a similar change to the Java repo 

```

        cordaBinDir = "${System.getProperty("user.home")}/$cordaBinariesDirectory"
        cordaCliBinDir = "${System.getProperty("user.home")}/$cordaCliBinariesDirectory"
```

**Comment by tlawson3** [comment]
Testing after commit f84196c added: https://ci02.dev.r3.com/job/QA/job/Craft_Tests/job/qa-regpack-csde/job/main/388/

**Review by marvincudjoe — DISMISSED** [review]

**Review by yarivus — APPROVED** [review]
LGTM

**Review by marvincudjoe — APPROVED** [review]

**Comment by tlawson3** [comment]
QA sign-off: https://r3-cev.atlassian.net/browse/CORE-16283?focusedCommentId=265324

**Review by christian-napoles-r3 — APPROVED** [review]

**Pull request merged**